src/support/random.string.generator.ts

```diff
@@ -0,0 +1,37 @@
+import cryptoRandomString from 'crypto-random-string';
+
+export const LOWER_CASE_LETTERS = 'abcdefghijklmnopqrstuvwxyz';
```

Comment:
Nice! We should get crypoRandomString to add these to their library too

Reply:
Yeah I agree! I thought it was strange that they didn't expose these. Instead, a lot of these are declared as a `type` and not a character set, but them implemented as a character set. 🤷🏼‍♂️